appinfo/info.xml

```diff
@@ -24,4 +24,9 @@
         <admin>OCA\Survey_Client\Settings\AdminSettings</admin>
         <admin-section>OCA\Survey_Client\Settings\AdminSection</admin-section>
     </settings>
+	<repair-steps>
```

Comment:
indentation is off

Reply:
actually the rest is of, its tabs vs. spaces